pages/linux/rtcwake.md

```diff
@@ -1,20 +1,20 @@
 # rtcwake
 
-> Enter a system sleep state until specified wakeup time relative to your bios clock.
+> Enter a system sleep state until specified wakeup time relative to your BIOS clock.
 
 - Show whether an alarm is set or not:
 
 `sudo rtcwake -m show -v`
 
-- Suspend to ram and wakeup after 10 seconds:
+- Suspend to RAM and wakeup after 10 seconds:
 
 `sudo rtcwake -m mem -s {{10}}`
 
 - Suspend to disk (higher power saving) and wakeup 15 minutes later:
 
 `sudo rtcwake -m disk --date +{{15}}min`
 
-- Freeze the system (more efficient than suspend-to-ram but linux > 3.9 required) and wakeup at a given date and time:
+- Freeze the system (more efficient than suspend-to-RAM but Linux kernel >3.9 required) and wakeup at a given date and time:
```

Comment:
My stance on this change changed — consider _"When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness."_ from [`CONTRIBUTING.md`](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md).
```suggestion
- Freeze the system (more efficient than suspend-to-RAM but 3.9 or greater of the Linux kernel is required) and wakeup at a given date and time:
```

Reply:
It said it must be newer than 3.9 before, but you wrote or must be 3.9 or newer on your one.
I think we need to verify which it's supposed to be, but overall thing your suggestion is nice.

Reply:
Hmm, I definitely didn't mean to change that, but I think it's actually correct!
From https://manned.org/rtcwake:
>The processes are frozen, all the devices are suspended and all the processors idled. ... (Available since Linux 3.9.)